### PR TITLE
Make footer section and Layout Component

### PR DESCRIPTION
**Pull request by revengemiroz**
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ module.exports = {
       colors: {
         primary: {
           400: '#ca322f'
-        }
+        },
+        lightRed: 'rgba(255,210,210,.46)'
```
Comment (rahuldahal) [inline tailwind.config.js:14]: this is fine for now.
But, I will fix this myself and update you about the changes

**Comment by rahuldahal** [comment]
Also, delete the `public/svgs/Footer/Logo.svg`(it's really messy). Just use placeholder text for now.

File: components/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        text: 'Line One Here',
+        link: '/url'
+      }
+    ]
+  }
+};
+
+function index(props) {
+  return (
+    <div className="bg-lightRed w-full h-full py-20">
```
[the remark this answers is not in the document]
Comment (revengemiroz) [reply components/Footer/index.js:89]: ok done

File: tailwind.config.js (hunk 1 above)
Comment (revengemiroz) [reply tailwind.config.js:14]: ok

**Review by rahuldahal — APPROVED** [review]

**Pull request merged**